docs/static/gamejam/jams/heist/rules.md

```diff
@@ -6,7 +6,7 @@ Does that mean your game has to be about robbing banks? Not at all! Your game co
 
 Some good interpretations of this theme:
 
-* A game where you control a rabbit stealing crops from a curmudgeony farmer
+* A game where you control a rabbit stealing crops from a curmudgeonly farmer
```

Comment:
it makes sense that it's curmudgeonly, but curmudgeony sounds way better in my head